### PR TITLE
First pass at a blog post

### PR DESCRIPTION
**Pull request by SteveALee**
Published so you can feedback before I gild it any more. It's mostly there but needs some tidy up and the code could possibly be split and comments added.

### CODE REVIEW

**Comment by SteveALee** [comment]
Pretty much complete now. Thanks

**Comment by kmaida** [comment]
Please incorporate the changes from the Gdoc and then we can do a final review and publish! Thank you :)

**Comment by kmaida** [comment]
FYI I will pull the code down and update to reflect changes from the Gdoc reviews.

**Comment by SteveALee** [comment]
@kmaida I missed your message, most likely in the conference mayhem. I'm now off for a meeting for the rest of the week so will try to do ASAP. I think I'll copy it all and convert to markdown - should be headings, links and code.

**Comment by SteveALee** [comment]
@kmaida I think that it's now all ready in the markdown fro a big squashy Merge. .Except your metadata of course.

Thanks again for all your help.

@nicosabena The code is easier to read now if you still wanted to check for latest best practices

File: _posts/2017-02-19-how-to-sign-in-with-google-and-then-access-their-APIs.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
+---
+layout: post
+title: "Using Severless Azure Functions with Auth0 to Access Google APIss"
+description: "TL;DR: Learn how Node.js backend code can access a Google API once a user logs in with Google via the Auth0 Lock widget."
+date: 2017-02-19 8:30
```
Comment (kmaida) [inline _posts/2017-02-19-how-to-sign-in-with-google-and-then-access-their-APIs.md]: @diegopoza Do we have a target publish date?

File: _posts/2017-02-19-how-to-sign-in-with-google-and-then-access-their-APIs.md [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+category: ??
+author:
+  name: Steve Lee
+  url: https://twitter.com/SteveALee
+  avatar: ??
```
Comment (kmaida) [inline _posts/2017-02-19-how-to-sign-in-with-google-and-then-access-their-APIs.md]: Please add a link to an avatar hosted on Gravatar here.

File: _posts/2017-02-19-how-to-sign-in-with-google-and-then-access-their-APIs.md [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+design:
+  bg_color: "#16214D"
+  image: https://cdn.auth0.com/blog/ultimate-abm-machine/ABM-logo.png
+tags:
+- ??
+related:
+- ??
```
Comment (kmaida) [inline _posts/2017-02-19-how-to-sign-in-with-google-and-then-access-their-APIs.md]:
```
related:
- 2016-06-28-building-serverless-apps-with-webtask
- 2015-10-29-extensibility-through-code-using-webtasks
```

File: _posts/2017-02-19-how-to-sign-in-with-google-and-then-access-their-APIs.md [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+category: ??
+author:
+  name: Steve Lee
+  url: https://twitter.com/SteveALee
+  avatar: ??
+design:
+  bg_color: "#16214D"
+  image: https://cdn.auth0.com/blog/ultimate-abm-machine/ABM-logo.png
+tags:
+- ??
```
Comment (kmaida) [inline _posts/2017-02-19-how-to-sign-in-with-google-and-then-access-their-APIs.md]:
```
tags:
- javascript
- serverless
- azure-functions
```

File: _posts/2017-02-19-how-to-sign-in-with-google-and-then-access-their-APIs.md [hunk 5]
```diff
@@ -0,0 +1,380 @@
+---
+layout: post
+title: "Using Severless Azure Functions with Auth0 to Access Google APIss"
+description: "TL;DR: Learn how Node.js backend code can access a Google API once a user logs in with Google via the Auth0 Lock widget."
+date: 2017-02-19 8:30
+category: ??
```
Comment (kmaida) [inline _posts/2017-02-19-how-to-sign-in-with-google-and-then-access-their-APIs.md]:
```
category: Technical guide
```

File: _posts/2017-02-19-how-to-sign-in-with-google-and-then-access-their-APIs.md (hunk 4 above)
Comment (SteveALee) [inline _posts/2017-02-19-how-to-sign-in-with-google-and-then-access-their-APIs.md]: +1

**Comment by SteveALee** [comment]
Meta data changes made (except the footer) just waiting for release date from @diegopoza

**Comment by SteveALee** [comment]
wahoo! :)

**Pull request merged**